index.html

```diff
@@ -172,7 +172,7 @@ <h2>Tonic</h2>
           </div>
         </div>
         <div class="tonic-desktop1">
-          <div class="imagez"><img src="images/snap3.png" alt="snap" /></div>
+          <div class="imagez"><img src="images/desk4.svg" alt="snap" /></div>
```

Comment:
Kindly update the alt text so that it conveys the purpose of the respective image as described above 😃